scrapy/__init__.py

```diff
@@ -30,13 +26,27 @@
 # Scrapy and Twisted versions
 __version__ = (pkgutil.get_data(__package__, "VERSION") or b"").decode("ascii").strip()
 version_info = tuple(int(v) if v.isdigit() else v for v in __version__.split("."))
-twisted_version = (_txv.major, _txv.minor, _txv.micro)
 
 
-# Ignore noisy twisted deprecation warnings
-warnings.filterwarnings("ignore", category=DeprecationWarning, module="twisted")
+def __getattr__(name: str):
+    if name == "twisted_version":
+        import warnings
+
+        # Ignore noisy twisted deprecation warnings
+        warnings.filterwarnings("ignore", category=DeprecationWarning, module="twisted")
```

Comment:
I think the reason for this line is to be executed before any Scrapy code, so that the user doesn't see those warnings. It was there before `from twisted import version` was even added.

Reply:
Pushed the rollback to original code.